controller/destination/listener_test.go

```diff
@@ -164,7 +164,7 @@ func TestEndpointListener(t *testing.T) {
 		expectedConduitNamespace := "conduit-namespace"
 		expectedPodDeployment := "pod-deployment"
 		expectedTlsIdentity := &pb.TlsIdentity_K8SPodIdentity{
-			PodIdentity:  "pod1.deployment.this-namespace.conduit-managed.conduit-namespace.svc.cluster.local",
+			PodIdentity:  "pod-deployment.deployment.this-namespace.conduit-managed.conduit-namespace.svc.cluster.local",
```

Comment:
I believe the `conduit-managed` fake service was removed in #1221, so I'm not sure fi thsi test is still correct?

Reply:
`conduit-managed` is hardcoded in the identity string, [here](https://github.com/runconduit/conduit/blob/master/pkg/k8s/labels.go#L162). So the test is correct based on what's in the code, but if the code needs to change, we can do that too.

Reply:
Ah, okay, it's not as important, then.